Servers/WindowServer/WSWindowSwitcher.cpp

```diff
@@ -56,13 +56,23 @@ void WSWindowSwitcher::on_key_event(const WSKeyEvent& event)
         }
         return;
     }
-    if (event.key() != Key_Tab) {
+
+    if (event.key() == Key_LeftShift || event.key() == Key_RightShift) {
+        return;
+    } else if (event.key() != Key_Tab) {
```

Comment:
No need for `else` after a `return`.

Reply:
Ah, so just `} if (event.key() != Key_Tab) {` will be good then?

Reply:
```cpp
}
if (event.key() != Key_Tab) {
    ...
}
```